package.json

```diff
@@ -38,6 +38,7 @@
   ],
   "pnpm": {
     "overrides": {
+      "flexsearch": "0.7.21",
```

Comment:
CMIIW, but I don't think this is package is necessary in the root `package.json`

```suggestion

```

Reply:
What this line in the root `package.json` under `pnpm/overrides` does is force the workspaces to use the specified `flexsearch` version. I didn't actually edit the `packages/nextra-theme-docs/package.json` line. The act of running `pnpm install` did so as a result of the override instruction.